### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
## Description

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/whalebird-desktop/commit/d4bfccb48d1f4d00fa2429a199b06b833617a25f#commitcomment-94273760

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/whalebird-desktop/commit/280d0439c8b358f63aefc0c136dfd683405e75e9

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,7 +42,7 @@
 - [#2116](https://github.com/h3poteto/whalebird-desktop/pull/2116) Use unicode-emoji-json instead of emojilib
 - [#2107](https://github.com/h3poteto/whalebird-desktop/pull/2107) build(deps-dev): Bump all-object-keys from 2.1.1 to 2.2.0
 - [#2105](https://github.com/h3poteto/whalebird-desktop/pull/2105) build(deps-dev): Bump eslint from 7.17.0 to 7.19.0
-- [#2115](https://github.com/h3poteto/whalebird-desktop/pull/2115) Use --node-env in webpack-cli instead of corss-env in pack command
+- [#2115](https://github.com/h3poteto/whalebird-desktop/pull/2115) Use --node-env in webpack-cli instead of cross-env in pack command
```
Comment (jsoref) [inline CHANGELOG.md:45]: Some projects don't like changing changelogs. Happy to drop this or anything else.

File: src/renderer/components/Preferences/General.vue [hunk 2]
```diff
@@ -13,7 +13,7 @@
         <el-switch id="sound_toot" v-model="sound_toot" active-color="#13ce66"> </el-switch>
       </el-form-item>
     </el-form>
-    <el-form class="timeline section" label-potision="right" label-width="302px">
+    <el-form class="timeline section" label-position="right" label-width="302px">
```
Comment (jsoref) [inline src/renderer/components/Preferences/General.vue:16]: Notable

File: src/renderer/components/TimelineSpace/Contents/FollowRequests.vue [hunk 3]
```diff
@@ -16,7 +16,7 @@ import User from '@/components/molecules/User.vue'
 import { ACTION_TYPES } from '@/store/TimelineSpace/Contents/FollowRequests'
 
 export default defineComponent({
-  name: 'folllow-requests',
+  name: 'follow-requests',
```
Comment (jsoref) [inline src/renderer/components/TimelineSpace/Contents/FollowRequests.vue:19]: Notable.

(Reviewdog also hates me, but I had to make a change in order to let it speak... That's a separate problem.)

File: src/renderer/components/molecules/User.vue [hunk 4]
```diff
@@ -41,7 +41,7 @@
       <el-button class="accept" link :title="$t('follow_requests.accept')" @click.stop.prevent="acceptRequest(user)">
         <font-awesome-icon icon="check" />
       </el-button>
-      <el-button class="reject" link :tilte="$t('follow_requests.reject')" @click.stop.prevent="rejectRequest(user)">
+      <el-button class="reject" link :title="$t('follow_requests.reject')" @click.stop.prevent="rejectRequest(user)">
```
Comment (jsoref) [inline src/renderer/components/molecules/User.vue:44]: notable

File: src/renderer/components/organisms/Notification.vue [hunk 5]
```diff
@@ -138,7 +138,7 @@ export default defineComponent({
     },
     focused: {
       type: Boolean,
-      defalt: () => false
+      default: () => false
```
Comment (jsoref) [inline src/renderer/components/organisms/Notification.vue:141]: notable

**Review by jsoref — COMMENTED** [review]
Most changes suggested by Google Sheets. All fault mine

**Comment by h3poteto** [comment]
Can you rebase this PR?

**Comment by jsoref** [comment]
@h3poteto done

File: src/renderer/components/TimelineSpace/Contents/FollowRequests.vue (hunk 3 above)
Comment (h3poteto) [reply src/renderer/components/TimelineSpace/Contents/FollowRequests.vue:19]: OK

**Review by h3poteto — APPROVED** [review]

**Pull request merged**